docs/current_docs/integrations/snippets/depot-hello.yml

```diff
@@ -0,0 +1,18 @@
+name: dagger
+on:
+  push:
+    branches: [main]
+
+jobs:
+  hello:
+    name: hello
+    runs-on: depot-ubuntu-22.04,dagger=0.13.7
+    steps:
+      -
+        name: Call Dagger Function
+        uses: dagger/dagger-for-github@v6
+        with:
+          version: "0.13.7"
+          verb: call
+          module: github.com/shykes/daggerverse/hello@v0.1.2
+          args: hello --greeting="bonjour" --name="monde"
```

Comment:
We are missing the following property from all `dagger/dagger-for-github` examples:

```yaml
engine-stop: false
```

We definitely **do not** want to stop this external Engine.

Reply:
Added